meet-data/svnl/03012015-penkkipunnerrus-p%C3%A4rnu-hpv/URL

```diff
@@ -0,0 +1 @@
+https://suomenvoimanostoliitto.fi/tulokset/kansalliset-kilpailut/03012015-penkkipunnerrus-p%C3%A4rnu-hpv
```

Comment:
This `MeetPath` is `svnl/03012015-penkkipunnerrus-p%C3%A4rnu-hpv` which requires double-escaping the `%%` signs, which is pretty ugly.

Also, the dates occur here in a non-standard order to the rest of the site. Since the `MeetPath` is the public name of the meet, that looks pretty inconsistent.

One option is to use the hash of the name, as in `cpu/cpu-reify-database` line 18. Python`s hash function is defined to be stable, and that produces pretty URLs.

A second option is to just use our default `1701`, `1702`, ..., which produces very short URLs.

Reply:
Thinking about it, I'd probably prefer `1701`, `1702`, etc.